newdle/client/src/components/summary/SummaryPage.js

```diff
@@ -9,35 +9,46 @@ import client from '../../client';
 import styles from './summary.module.scss';
 
 export default function SummaryPage() {
-  const [submitting, setSubmitting] = useState(false);
-  const [error, setError] = useState('');
   const [finalDate, setFinalDate] = useState(null);
   const newdle = useSelector(getNewdle);
   const dispatch = useDispatch();
+  const [_sendResultEmails, mailSending, mailError, sendMailResponse] = client.useBackend(
+    client.sendResultEmails
+  );
+  const [_setFinalDate, submitting, finalDateError] = client.useBackend(client.setFinalDate);
 
   const update = async () => {
-    setError('');
-    setSubmitting(true);
-    let updatedNewdle;
-    try {
-      updatedNewdle = await client.setFinalDate(newdle.code, finalDate);
-    } catch (exc) {
-      setSubmitting(false);
-      setError(exc.toString());
-      return;
+    const updatedNewdle = await _setFinalDate(newdle.code, finalDate);
+    if (updatedNewdle) {
+      dispatch(updateNewdle(updatedNewdle));
     }
-    dispatch(updateNewdle(updatedNewdle));
-    setSubmitting(false);
+  };
+
+  const sendResultEmails = async () => {
```

Comment:
you can just call `_sendResultEmails(newdle.code);` directly' no need to async/await if you don't actually do something with the result (or once the call finished)